settings_dialog.py

```diff
@@ -25,46 +27,90 @@ class CroProSettingsDialog(QDialog):
     def __init__(self, *args, **kwargs) -> None:
         QDialog.__init__(self, *args, **kwargs)
         disable_help_button(self)
+        self.tab_view = QTabWidget()
         self.checkboxes = make_checkboxes()
-        self.tag_edit = QLineEdit(config.tag_original_notes)
+        self.tag_edit = QLineEdit(config.exported_tag)
         self.max_notes_edit = CroProSpinBox(min_val=10, max_val=10_000, step=50, value=config.max_displayed_notes)
         self.hidden_fields = ItemEditBox("Hidden fields", initial_values=config.hidden_fields)
         self.web_timeout_spinbox = CroProSpinBox(min_val=1, max_val=999, step=1, value=config.timeout_seconds)
-        self.button_box = QDialogButtonBox(BUT_OK | BUT_CANCEL)
+        self.button_box = QDialogButtonBox(
+            (BUT_HELP | BUT_OK | BUT_CANCEL) if config.show_help_buttons else (BUT_OK | BUT_CANCEL)
+        )
```

Comment:
Do we really need to be able to disable the help button? If it's always visible, it won't harm anybody.